virtualenv_tools.py

```diff
@@ -304,7 +360,9 @@ def main(argv=None):
         return 1
 
     try:
-        venv = _get_original_state(args.path)
+        is_pypy = os.path.isdir(os.path.join(args.path, 'lib_pypy'))
+        venv = _get_original_state_pypy(args.path) \
+            if is_pypy else _get_original_state(args.path)
```

Comment:
`is_pypy` seems like part of the original state, and as such belongs *inside* the get-original-state function.

Did you know: the cyclomatic complexity of a try-block increases by one for each *operation* it contains.

- [ ] fixed